PSKoans/Koans/dbatools/AboutNewDatabases.ps1

```diff
@@ -0,0 +1,194 @@
+            Collation = 'SQL_Latin1_General_CP1_CI_AS'
+            Owner = $ENV:USERNAME
+            LastFullBackup = (Get-Date '0001-01-01')
+            LastDiffBackup = (Get-Date '0001-01-01')
+            LastLogBackup = (Get-Date '0001-01-01')
+        }
+    } -ParameterFilter {
+        $_.SqlInstance -eq 'localhost' -and
+        $_.RecoveryModel -eq 'Full'
+    }
+
+    <#
+        New-DbaDatabase, while having a few different parameters, requires only that you have an instance
+        of SQL Server that you can connect to and it can create a database on.
+
+        Complete the below to create a new database on localhost.
+    #>
+    $NewDatabase = New-DbaDatabase -SqlInstance '__'
+    $NewDatabase.ComputerName | Should -Be 'localhost'
+
+    <#
+        While only specifying the server name is convenient, it is rare that we want a randomly named
+        database. Most of the time we want to specify the database name(s) to be created.
+
+        Luckily, New-DbaDatabase allows us to do that!
+    #>
+    $NewSpecifiedDatabase = New-DbaDatabase -SqlInstance localhost -Name '____', '____'
+    $NewSpecifiedDatabase.Name | Should -Be 'RandomIsGood', 'SpecificIsBetter'
+
+    <#
+        If you're interacting with databases regularly then you more than likely have a list of
+        scripts that you run often enough to save. You can save these scripts as files or you can
+        save them to a database on each server.
+
+        If only there was a way to create a database on your Dev, Val, and Prod servers that you
+        can save these scripts to...
+    #>
+    $NewScriptsDatabase = New-DbaDatabase -SqlInstance '____', '____', '____' -Name DBScripts
+    $NewScriptsDatabase.SqlInstance | Should -Be 'Dev', 'Val', 'Prod'
+
+    <#
+        When new databases get created they get created with a recovery model. They take this
+        recovery model from the system database model.
+
+        New-DbaDatabase allows you to specify a recovery model that you want the new database to
+        have. This recovery model does not need to be what the model database is set to.
+
+        Finish the below block of code to create a new database with the FULL recovery model.
+    #>
+    $NewRecoveryModelDatabase = New-DbaDatabase -SqlInstance localhost -RecoveryModel '____'
+    $NewRecoveryModelDatabase.RecoveryModel | Should -Be 'Full'
+
+    <#
+        There is also an option to specify where to place the database data and log files. By using the parameters
+        -DataFilePath and -LogFilePath, you can specify where to place these files.
+    #>
+    $NewFilePathDatabase = New-DbaDatabase -SqlInstance localhost -Name FilePathTese -DataFilePath '____' -LogFilePath '___'
+    $DataPath = $NewFilePathDatabase.FileGroups.Files
+    $LogPath = $NewFilePathDatabase.LogFiles
+
+    (Split-Path -Path $DataPath).FileName | Should -Be 'E:\DATA'
+    (Split-Path -Path $LogPath).FileName | Should -BE 'E:\LOG'
+}
```

Comment:
Might also be a good idea to go through and ensure that all your files end in a newline. it's a Unix compatibility thing mainly, but I have tests hinging on that so 😁